### PR TITLE
Assignment2

### PR DESCRIPTION
**Pull request by RichardTeller**

### CODE REVIEW

File: SecretSanta/test/SecretSanta.Import.Tests/UserGiftImporterTests.cs [hunk 1]
```diff
@@ -13,15 +15,15 @@ public class UserGiftImporterTests
         [TestInitialize]
         public void TestInitialize()
         {
-            TestFilePath = Environment.CurrentDirectory + "\\TestFile.txt";
+            TestFilePath = System.IO.Path.GetTempFileName();
```
Comment (BHoward85) [inline SecretSanta/test/SecretSanta.Import.Tests/UserGiftImporterTests.cs]: I had some trouble with the file placement and access. Now seeing how you used the Temp File and ease of clean up. Very good.

File: SecretSanta/src/SecretSanta.Import/UserGiftImporter.cs [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+                    return new string[] { headerType, firstName, lastName };
+                }
+                else
+                {
+                    string firstName = header.Substring(0, header.IndexOf(" "));
+                    string lastName = header.Substring(header.IndexOf(" ") + 1);
+
+                    return new string[] { headerType, firstName, lastName };
+                }
```
Comment (BHoward85) [inline SecretSanta/src/SecretSanta.Import/UserGiftImporter.cs]: Have you tested all the cases that should there be a missing first name, last name, or that there is no name past the ":" in the header? I just letting you know that so far I can not see that you are checking for missing parts.

Comment (RichardTeller) [reply SecretSanta/src/SecretSanta.Import/UserGiftImporter.cs]: I don't believe we are supposed to check for extraneous things like that.  Mark said the point of the assignment was to practice TDD.  In the assignment instructions it says we are given a valid file.  The only things we need to check for in terms of format is which of the 2 header formats is the header, and the gifts can contain blank lines or if its not blank then its a valid gift name.

**Comment by MarkMichaelis** [comment]
Overall I think you did a nice job!
The API design could be improved.  You never actually imported the data you parsed which was seemingly strange?

**Pull request closed without merging**